index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my-portfolio</title>
+</head>
+<body>
+    <section class="container">
+        <header>
+            <nav>
+                <a href="#">
+                    <img class="my-logo" src="images/my-logo.png" alt="logo">
+                </a>
+                <a href="#">
+                    <img class="menu" src="images/union.png" alt="union">
+                </a>
+            </nav>
+        </header>
+        <section class="main-section">
+            <br>
+            <h1 class="main-text">I’m Godanah
+                Glad to see you!</h1>
+            <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <br>
+            <p>LET'S CONNECT</p>
```

Comment:
- [ ] [OPTIONAL] Please, try to use the margin/padding according to the Figma design you selected in order to make more professional your portfolio 😉 

Figma design |  Yours

![image](https://user-images.githubusercontent.com/60164239/129457548-2c4a1727-17a6-4fa3-922f-9c2925e5ada1.png)